roles/geo_replication/tasks/georep_session_stop.yml

```diff
@@ -0,0 +1,11 @@
+# Stop geo-replication session
+- name: Stop the geo-rep session
+  geo_rep:
+     action: stop
+     mastervol: "{{ gluster_features_georep_mastervol }}"
+     slavevol: >
+       "{{ gluster_features_georep_slavenodes[0] }}:{{ gluster_features_georep_slavevol }}"
+     force: true
+     georepuser: "{{ georepuser }}"
+  run_once: true
+  when: gluster_features_georep_slavenodes is defined
```

Comment:
We don't need a separate task file for stop. This has to be done as part of cleanup. So, use this in your cleanup implementation.

Reply:
ack.